### ISSUE TITLE
how do I remove an association?

### ISSUE THREAD

**Issue by exekutive** [Lord-Kamina/SwiftDefaultApps#54]
I had previously associated .nfo files with an application. I want these files to be just unrecognized generic files again, without any UTI entries.
Can this utility do that?

**Comment by Lord-Kamina** [comment]
I don't think that's possible. The only way for LaunchServices to forget a UTI is for all apps referring to it to go.

**Comment by Alkenso** [comment]
I think that would be a good option to 'erase' all assotiations with particular schema or UTI

**Comment by SamBeckett97** [comment]
Yup, really need a way to delete a URI entry and clear extraneous associated applications.

**Comment by rosmcmahon** [comment]
+1

**Comment by rosmcmahon** [comment]
still no joy. this is totally messing up certain browser links for me. how to manually remove a custom URL ??

**Comment by rrhill** [comment]
Also would like to uncheck to delete the assignment.

**Comment by rosmcmahon** [comment]
I installed a browser I don't use. Made the URL link open in this browser, then uninstalled that browser. This worked!

I also then uninstalled this extension.
